server/api/cartItem.js

```diff
@@ -44,10 +70,39 @@ router.get('/:cartItemId?', (req, res, next) => {
 
 router.put('/:cartItemId', (req, res, next) => {
   const { cartItemId } = req.params;
-  CartItem.update({ ...cartItem, ...req.body }, { where: { id: cartItemId }, returning: true })
-  CartItem.findOne({ where: { id: cartItemId } })
-  // finish this
+  CartItem.findOne({
+    where: {
+      id: cartItemId
+    }
+  })
+    .then(cart => {
+      cart.update({ ...cart, ...req.body }, { returning: true })
```

Comment:
Eliot had mentioned in class that PUT or POST with ...req.body is a security vulnerability. You should pull out specific properties